zipkin/src/main/java/zipkin/TraceKeys.java

```diff
@@ -99,6 +99,12 @@
    */
   public static final String HTTP_RESPONSE_SIZE = "http.response.size";
 
+  /**
+   * The query executed for JDBC call.  Ex. "select * from customers where id = ?"
```

Comment:
I know I suggested this.. :) but wondering if we shouldn't say "sql.query" instead of "jdbc.query"
That way, the annotation would be valid for non-java